tasks/rsync-deploy.yml

```diff
@@ -15,13 +27,6 @@
     archive: yes
   delegate_to: "{{ inventory_hostname }}"
 
-# Ensure symlinks target paths is absent
-- name: ANSISTRANO | Ensure shared paths targets are absent
-  file:
-    state: absent
-    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}/{{ item }}"
-  with_items: "{{ ansistrano_shared_paths }}"
-
 # Symlinks shared paths
```

Comment:
That would reduce downtime during deployment (first all symlinks are removed, than all symlinks are readded).

Would be better to remove and add each symlink to shared one after one.

I've reverted that change because if shared folders are changing (eg. a existing folder is now a shared folder) the deployment failes because ansible cannot create the symlink.
I'm still trying to find a good solution.